client/indexer-db/src/models/bucket.rs

```diff
@@ -80,4 +80,15 @@ impl Bucket {
             .await?;
         Ok(bucket)
     }
+
+    pub async fn get_by_onchain_bucket_id<'a>(
```

Comment:
Not specific to this PR, but are we not storing the Bucket root?

Reply:
Not in this PR, no. I was planning to add root for both buckets and BSPs in another PR.

Is this fine?

Reply:
Yeah that's fine. But we have to be aware that until then we want be able to use the main functionality of our indexer.